tasks/pipeline-pip-deps.yml

```diff
@@ -9,12 +9,13 @@
     - "archivematica-mcp-client"
     - "archivematica-mcp-server"
 
-- name: "virtualenv | Create environment and install requirements"
+- name: "Create virtualenv and install pip-tools"
   pip:
     virtualenv: "{{ archivematica_src_am_virtualenv }}"
-    requirements: "{{ archivematica_src_dir }}/archivematica/{{ 'requirements-dev.txt' if is_dev else 'requirements.txt' }}"
+    name: "pip-tools"
+  tags: "amsrc-ss-pydep"
```

Comment:
I think this tag is [Storage Service specific](https://github.com/artefactual-labs/ansible-archivematica-src/blob/5c0a676fb8f2793d050748c959dfab12c7d1937a/README.md#tags) and this is a pipeline playbook.

Reply:
Ops, thank you, fixed!